lib/util/mailables.js

```diff
@@ -113,13 +113,22 @@ async function writePDF(formData, imageData, otpConfig) {
 
   const tableData = {
     headers: ['Item', 'Quantity'],
-    rows: mailables.map((mailable) => {
-      let { largeFormat, largePrint, name, quantity } = mailable
-      if (largePrint && largeFormat) {
-        name += ' (LARGE PRINT)'
-      }
-      return [name, quantity]
-    })
+    rows: mailables
+      .map((mailable) => {
+        let { largeFormat, largePrint, name, quantity, smallFormat } = mailable
+        const rows = []
+
+        if (smallFormat) {
+          rows.push([name, quantity])
+        }
+
+        if (largePrint && largeFormat) {
+          name += ' (LARGE PRINT)'
+          rows.push([name, quantity])
+        }
+        return rows
+      })
+      .flat()
```

Comment:
Nitpicky, but could just change the above `map` to `flatMap`

Reply:
This is much cleaner thanks!